profile/README.md

```diff
@@ -55,4 +55,4 @@ You can also financially support the project by contributing on [LocalStack's Op
 
 ## Join our team! 🤝
 
-We are looking for top-notch contributors to LocalStack. If you are interested in helping us become the world’s leading platform to support efficient dev & test loops for cloud applications, check out our [current open positions](https://localstack.cloud/jobs/). We can't wait to meet you!
+We are looking for top-notch contributors to LocalStack. If you are interested in helping us become the world’s leading platform to support efficient dev & test loops for cloud applications, check out our [current open positions]([https://localstack.cloud/jobs/](https://www.localstack.cloud/career-s)). We can't wait to meet you!
```

Comment:
Thanks for making us aware of the broken link and for taking the time to create a PR to adjust the link!
However, in the meantime we slightly fixed the link (in fact `/jobs` now also works for backwards compatibility, but it would be great to adjust the link to the latest URL), and it would be great to also adjust the link text:
```suggestion
We are looking for top-notch contributors to LocalStack. If you are interested in helping us become the world’s leading platform to support efficient dev & test loops for cloud applications, check out our [current open positions]([https://localstack.cloud/careers/](https://www.localstack.cloud/careers)). We can't wait to meet you!
```

Once these changes have been incorporated, we can merge this right a way :)